testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -382,42 +382,40 @@ class PlatformViewSurroundingLayersScenario extends Scenario
   void onBeginFrame(Duration duration) {
     final SceneBuilder builder = SceneBuilder();
 
-    // a square platform view from (99.5, 99.5) to (200.5, 200.5)
-    builder.pushOffset(99.5, 99.5);
+    builder.pushOffset(101, 101);
     addPlatformView(
       id,
-      width: 101,
-      height: 101,
+      width: 100,
+      height: 100,
       dispatcher: view.platformDispatcher,
       sceneBuilder: builder,
     );
-
     builder.pop();
 
     final PictureRecorder recorder = PictureRecorder();
     final Canvas canvas = Canvas(recorder);
 
     // Rect at the left of platform view
     canvas.drawRect(
-      const Rect.fromLTWH(0, 99.5, 99.5, 99.5),
+      const Rect.fromLTWH(0, 101, 101, 101),
       Paint()..color = const Color(0x22FF0000),
     );
 
     // Rect at the right of platform view
     canvas.drawRect(
-      const Rect.fromLTWH(200.5, 99.5, 99.5, 99.5),
+      const Rect.fromLTWH(201, 101, 101, 101),
```

Comment:
Doesn't the PV end at 200.5? Shouldn't this be "200" so as to create an overlap pixel?

Reply:
reverted back to `200.5`. I want to have the 2 perfectly edge to edge to mimic the original behavior.